src/views/LoginForm/loginFormConstants.json

```diff
@@ -0,0 +1,12 @@
+{ "loginFormInputs" : [
```

Comment:
you don't need to assign it as a value. just set it as an array immediately.

```json
[
   { "a": 123 },
   { "a": 321 }
]
```

Reply:
The importing it from `js` will simply be 

`import data from 'yourJson.json'`